.travis.yml

```diff
@@ -1,30 +1,35 @@
 language: cpp
+compiler: gcc
 sudo: true
-services: docker
```

Comment:
The problem with docker IIRC is that you need sudo access to the image in order to install prerequisites.